packages/cli/src/scaling/pubsub/pubsub-handler.ts

```diff
@@ -58,4 +65,34 @@ export class PubSubHandler {
 	}
 
 	// #endregion
+
+	// #region Worker process
+
+	private setupWorkerHandlers() {
+		this.setupHandlers({
```

Comment:
Do we have a way to detect uncaught messages? If not, it's not necessary for this to be added now, just might help with debugging any issues later.

Reply:
Good idea, capturing separately: https://linear.app/n8n/issue/PAY-2065